supported_parts.md

```diff
@@ -160,6 +160,7 @@
 - ADXL314
 - ADXL345
 - ADXL355
+- ADXL380
```

Comment:
shouldn't this contain ADXL382 as well ?

Reply:
added adxl382 as well